varya/assets/sass/blocks/file/_style.scss

```diff
@@ -17,12 +17,10 @@
 	}
 
 	.entry-content &.alignleft {
-		@extend %responsive-alignright;
-		margin-right: var(--global--spacing-horizontal);
+		@extend %responsive-alignleft;
```

Comment:
Why is it that this block stills need the alignment rules here? Shouldn't the alignment utility classes take care of it? Same question for the pullquote and video blocks.

Reply:
This might be able to be removed too. Need to test it with Gutenberg 8.0 to be sure, but at the time I added this, it was still necessary.

Reply:
Tested, and these are still necessary.